### DISCUSSION TITLE
Taxonomy loop in Entry template

### DISCUSSION THREAD

**Discussion by tomasg-lt** [statamic/cms#5891] [Q&A]
I have a sidebar on the Collection Entry page with a list of all specific taxonomies. Like:

`{{ taxonomy from="hardware" }}
           <li><a href="{{ url }}" class="category">{{ title }}</a></li>
{{ /taxonomy }}`

The Entry belongs to one of the taxonomies from the list. Like:

`hardware:
- CPU`

I want to add an additional class to the list item if my currently open Collection Entry belongs to the same term. How shoud I do it properly in that taxonomy loop?

Thanks!

**Comment by edalzell** [comment]
what do you mean "currently open collection entry"?
